rafthttp/http.go

```diff
@@ -163,7 +162,7 @@ func (h *streamHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	case path.Join(RaftStreamPrefix, string(streamTypeMessage)):
 		t = streamTypeMessage
 	default:
-		log.Printf("rafthttp: ignored unexpected streaming request path %s", r.URL.Path)
+		plog.Warningf("ignored unexpected streaming request path %s", r.URL.Path)
```

Comment:
agreed. this could happen when higher version dial the lower version. because we have checked version compatibility before, it cannot be user's fault.